ui/src/pages/home/HomePage.js

```diff
@@ -20,41 +24,123 @@ const HomePage = () => {
-            <Box flex="1">
-              <Heading as="h1" fontSize="40px">
-                Le {PRODUCT_NAME}
-              </Heading>
-              <Text fontSize="beta" color="grey.800" marginTop="4w">
-                Mettre à disposition des <strong>différents acteurs</strong> <br />
-                les <strong>données clés</strong> de l&apos;apprentissage en <strong>temps réel</strong>
-              </Text>
-            </Box>
-            <Image src={dashboardIllustration} alt="illustration tableau de bord" paddingBottom="3w" />
-          </Flex>
-          <HStack spacing="3w" _hover={{ cursor: "pointer" }}>
-            <LinkCard linkHref={NAVIGATION_PAGES.Login.path}>
-              Vous êtes une{" "}
+      <Section withShadow paddingY="6w" color="grey.800" background="#F6F6F6">
+        <Flex>
+          <Box flex="1" alignSelf="center">
+            <Heading as="h1" fontSize="40px">
+              Le {PRODUCT_NAME}
+            </Heading>
+            <Text fontSize="gamma" color="grey.800" marginTop="4w">
+              Service public pour visualiser <strong>les effectifs d’apprentis en temps réel</strong> dans les centres
+              et organismes de formation. Il permet ainsi aux pouvoirs publics de{" "}
               <strong>
-                institution ou une organisation <br />
-                professionnelle{" "}
-              </strong>
-              (OPCO, branche, etc...)
-            </LinkCard>
-            <LinkCard linkHref={NAVIGATION_PAGES.OrganismeFormation.path}>
-              Vous êtes un{" "}
+                piloter <br />
+                la politique de l’apprentissage
+              </strong>{" "}
+              nationalement et dans les territoires.
+            </Text>
+          </Box>
+          <GraphsAndStatistics />
+        </Flex>
+        <HStack marginTop="4w" spacing="3w" _hover={{ cursor: "pointer" }}>
+          <Box
+            as={NavLink}
+            to={NAVIGATION_PAGES.OrganismeFormation.path}
+            border="1px solid"
+            borderColor="bluefrance"
+            padding="4w"
+            width="50%"
+          >
+            <Flex>
+              <School />
+              <Box alignSelf="center" marginLeft="2w">
+                <Text fontSize="gamma">
+                  Vous êtes un{" "}
+                  <strong>
+                    organisme de formation <br />
+                    en apprentissage
+                  </strong>
+                </Text>
+              </Box>
+            </Flex>
+            <Text marginTop="2w">
+              Permettez le pilotage de la politique de l&apos;apprentissage en temps réel en donnant de la visibilité
+              sur vos effectifs d&apos;apprentis.
+            </Text>
+            <Box marginTop="4w">
+              <ArrowLink title="Découvrir" />
+            </Box>
+          </Box>
+          <Box width="50%" as={NavLink} to={"/login"} border="1px solid" borderColor="bluefrance" padding="4w">
```

Comment:
Tu peux remplacer le lien vers la constante : 

```suggestion
          <Box width="50%" as={NavLink} to={NAVIGATION_PAGES.Login.path} border="1px solid" borderColor="bluefrance" padding="4w">
```